packages/atomic/src/components/facets/atomic-category-facet/readme.md

```diff
@@ -10,41 +10,41 @@ A hierarchical category facet component. It is displayed as a facet in desktop b
-| `field`               | `field`                | Specifies the index field whose values the facet should use                                                                                                                                                    | `string`                          | `''`            |
+| `facetId`             | `facet-id`             | Specifies a unique identifier for the facet.                                                                                                                                                                   | `string`                          | `''`            |
+| `field`               | `field`                | Specifies the index field whose values the facet should use.                                                                                                                                                   | `string`                          | `''`            |
 | `filterByBasePath`    | `filter-by-base-path`  | Whether to use basePath as a filter for the results.                                                                                                                                                           | `boolean`                         | `true`          |
-| `label`               | `label`                | The non-localized label for the facet                                                                                                                                                                          | `string`                          | `'noLabel'`     |
+| `label`               | `label`                | The non-localized label for the facet.                                                                                                                                                                         | `string`                          | `'noLabel'`     |
 | `numberOfValues`      | `number-of-values`     | The number of values to request for this facet. Also determines the number of additional values to request each time this facet is expanded, and the number of values to display when this facet is collapsed. | `number`                          | `8`             |
-| `sortCriteria`        | `sort-criteria`        | The sort criterion to apply to the returned facet values. Possible values are 'alphanumeric', and 'occurrences''.                                                                                              | `"alphanumeric" \| "occurrences"` | `'occurrences'` |
+| `sortCriteria`        | `sort-criteria`        | The sort criterion to apply to the returned facet values. Possible values are `alphanumeric`, and `occurrences`.                                                                                               | `"alphanumeric" \| "occurrences"` | `'occurrences'` |
```

Comment:
I believe the types here are incorrect, but wasn't sure how far to drill down. Should the type of `sort-criteria` in this table reduce to primitive types?

Reply:
Yep, a developer would have to specify one of these values, so seeing them in the documentation is helpful.

Reply:
OK. I'm going to adjust this on the docs side.